chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.tsx

```diff
@@ -150,8 +154,28 @@ const ChatFeed: React.FC = () => {
     setIsRightPanelCollapsed(!isRightPanelCollapsed);
   const collapseLeftPanel = () =>
     setIsLeftPanelCollapsed(!isLeftPanelCollapsed);
-  const insertImage = () => console.log('Image has been inserted');
-  const attachFile = () => console.log('File has been attached');
+  const attachFile = () => {
+    fileInputRef.current?.click();
+  };
+
+  const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[e.target.files.length - 1];
```

Comment:
can you put e.preventDefault(); right above this line.

Reply:
I dont think its needed, works ok